### PR TITLE
build: bump to angular 6.0.0-rc.1 and rxjs 6.0.0-rc.0

### PR DESCRIPTION
**Pull request by crisbeto**
Bumps to the latest Angular and RxJS RCs and fixes a compilation error in the postinstall script.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
     "@types/glob": "^5.0.33",
     "@types/gulp": "3.8.32",
     "@types/hammerjs": "^2.0.35",
-    "@types/jasmine": "^2.6.0",
+    "@types/jasmine": "2.2.34",
```
Comment (crisbeto) [inline package.json]: @jelbourn I've bumped this one down temporarily due to a compilation error coming in from core. I've submitted https://github.com/angular/angular/pull/23110 to fix it.

File: package.json [hunk 2]
```diff
@@ -26,14 +26,14 @@
...
-    "@angular/platform-browser": "6.0.0-rc.0",
+    "@angular/animations": "6.0.0-rc.1",
+    "@angular/common": "6.0.0-rc.1",
+    "@angular/compiler": "6.0.0-rc.1",
+    "@angular/core": "6.0.0-rc.1",
+    "@angular/forms": "6.0.0-rc.1",
+    "@angular/platform-browser": "6.0.0-rc.1",
     "core-js": "^2.4.1",
-    "rxjs": "6.0.0-beta.1",
+    "rxjs": "6.0.0-rc.0",
```
Comment (crisbeto) [inline package.json]: @jelbourn I'm not completely sure why Bazel is complaining about not being able to find rxjs. Do you have any ideas?

Comment (CaerusKaru) [reply package.json]: You need to bump the Bazel rules for nodejs and typescript in WORKSPACE. See [here](https://github.com/angular/flex-layout/blob/4864f2b20324e214038bf7cbc57b976ade04e58d/WORKSPACE) for the correct versions

You may also need to add `@rxjs//operators` to some BUILD files if Bazel still complains

Comment (CaerusKaru) [reply package.json]: One more thing, you also need to add "rxjs" to the list of node_modules in the top-level BUILD file. Then the only errors left should be the operators import.

Comment (CaerusKaru) [reply package.json]: After pulling and running locally, the following are the only BUILD files that need to be modified: cdk/a11y, cdk/overlay, and cdk/tree, and only the test sources need `@rxjs//operators` to build correctly

Comment (crisbeto) [reply package.json]: Thanks for looking into it, but I'm not sure that's the case. I tried adding it to all of them and it ended up erroring on the `cdk/portal`.

Comment (CaerusKaru) [reply package.json]: You never added rxjs to the list of node_modules in the top-level BUILD

Comment (CaerusKaru) [reply package.json]: You never added rxjs to the top-level BUILD node_modules

Comment (CaerusKaru) [reply package.json]: See [here](https://github.com/angular/flex-layout/pull/652/files#diff-0ea22445d900db774c8deb00f4a4df55R24) for reference

Comment (crisbeto) [reply package.json]: Looks like that did it, although it still feels weird to add `rxjs/operators` when it was complaining about `rxjs/index`.

Comment (CaerusKaru) [reply package.json]: The `rxjs/index` issue was resolved by adding the `rxjs` to top-level BUILD and updating the bazel rules. The operators are to reflect the latest changes in structure for RxJS RC

File: WORKSPACE [hunk 3]
```diff
@@ -27,10 +27,10 @@ sass_repositories()
 
 # Add TypeScript rules
 http_archive(
-  name = "build_bazel_rules_typescript",
-  url = "https://github.com/bazelbuild/rules_typescript/archive/0.11.1.zip",
-  strip_prefix = "rules_typescript-0.11.1",
-  sha256 = "7406bea7954e1c906f075115dfa176551a881119f6820b126ea1eacb09f34a1a",
+    name = "build_bazel_rules_typescript",
```
Comment (josephperrott) [inline WORKSPACE]: remove extra indenting from parameters.

The rest of the file only uses 2 spaces

Comment (crisbeto) [reply WORKSPACE]: Done.

File: BUILD.bazel [hunk 4]
```diff
@@ -26,6 +26,7 @@ filegroup(
     "protobufjs",
     "protractor",
     "reflect-metadata",
+    "rxjs",
```
Comment (jelbourn) [inline BUILD.bazel]: rxjs is actually not _supposed_ to be here because it has its own bazel rules. Can you share the error you were getting?

Comment (crisbeto) [reply BUILD.bazel]: I don't have the exact error, but it was something along the lines of "Can't find module `rxjs/index`".

Comment (jelbourn) [reply BUILD.bazel]: Chatted w/ Alex, this seems like an issue w/ the ngsummary files being generated, we're figuring out the right way for core to fix this

**Pull request merged**